scripts/s3/migrate_folder_language.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import sys
+import time
+
+from modularodm import Q
+
+from framework.mongo import database
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.addons.base.exceptions import InvalidAuthError, InvalidFolderError
+from website.addons.s3.utils import get_bucket_location_or_error
+from website.addons.s3.settings import BUCKET_LOCATIONS
+from website.addons.s3.model import S3NodeSettings
+from scripts import utils as script_utils
+
+
+logger = logging.getLogger(__name__)
+
+
+def migrate(dry_run=False):
+    bucket_name_location_map = {}
+    sns_collection = database['s3nodesettings']
+    logger.info('Migrating all {} S3NodeSettings documents: {}'.format(
+        sns_collection.count(), [s['_id'] for s in sns_collection.find()]
+    ))
+    for document in sns_collection.find():
+        sns_collection.find_and_modify(
+            {'_id': document['_id']},
```

Comment:
We will have to add a blocker issue to the go-nogo ticket to ensure that the migration is tested on Production data while this is going through QA. If it is at all unreliable, we may consider refactoring so that it's continuable, but otherwise should be fine.